kotlin-analysis-api/testData/getSubPackages.kt

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Tests under `kotlin-analysis-api` are forked version of those in `test-utils`. They are (ideally) only used when there is a behavior difference between KSP1 and KSP2. It doesn't seem to be necessary in this case.